.circleci/config.yml

```diff
@@ -28,16 +28,10 @@ jobs:
         command: |
           docker build -t $REGISTRY_URL:$CIRCLE_SHA1 --target base .
     - run:
-        name: "Niftany"
-        background: true
+        name: "Linting"
```

Comment:
I think the Niftany name is more appropriate since it should be running the `bundle exec niftany` command.